request_logging/middleware.py

```diff
@@ -66,6 +67,11 @@ def __init__(self, get_response=None):
 
         self.log_level = getattr(settings, SETTING_NAMES['log_level'], DEFAULT_LOG_LEVEL)
         self.http_4xx_log_level = getattr(settings, SETTING_NAMES['http_4xx_log_level'], DEFAULT_HTTP_4XX_LOG_LEVEL)
+        self.sensitive_headers = getattr(settings, SETTING_NAMES['sensitive_headers'], [])
```

Comment:
This all looks great and thanks for doing this; should we set some sensible defaults here so people who never even consider this get some benefit?

Reply:
Updated the codes with default sensitive headers `HTTP_AUTHORIZATION` and `HTTP_PROXY_AUTHORIZATION`.  If you think some other headers makes sense, please do suggest.